Public/Src/Cache/DistributedCache.Host/Configuration/DistributedContentSettings.cs

```diff
@@ -40,6 +40,11 @@ public class DistributedContentSettings
         [JsonConstructor]
         private DistributedContentSettings()
         {
+            if (MemoizationExpiryTimeMinutes == 0)
+            {
+                // Sync memoization with content unless specified.
+                MemoizationExpiryTimeMinutes = ContentHashBumpTimeMinutes;
+            }

```

Comment:
ContentHashBumpTimeMinutes is typically set after construction. This logic should be in the property getter with a backing field.